#request
float64 latitude  #/*!< unit: rad */
float64 longitude #/*!< unit: rad */
---
#response
bool result